Resources/Textures/DeltaV/Objects/Consumable/Food/scrambledeggs.rsi/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation and modified by Swept at https://github.com/tgstation/tgstation/commit/40d75cc340c63582fb66ce15bf75a36115f6bdaa",
```

Comment:
Who drew the scrambled egg sprite? Is it also from tgstation and modified by Swept?

Reply:
Nah my girlfriend made it. I didn't know what to put in the copyright. I can change it to her Alias if that is more appropriate.

Reply:
Yeah you can put anything in the copyright field, whether it’s alias, URL, or both. Just make sure to include the preexisting copyright for the bowl.

Reply:
Got it. Which i am gonna guess is tgstation.